dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/IndexStats.java

```diff
@@ -0,0 +1,34 @@
+package com.dotcms.content.elasticsearch.business;
+
+import org.elasticsearch.common.unit.ByteSizeValue;
+
+public class IndexStats {
+
+    private String indexName;
+    private int documentCount;
+    private int size;
+    private String prettySize;
+
+    public IndexStats(String indexName, int documentCount, int size) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'size' is not assigned and could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=3712629)